src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -6837,25 +6844,33 @@ public static IAsyncEnumerable<Task> WhenEach(params Task[] tasks)
         }
 
         /// <inheritdoc cref="WhenEach(Task[])"/>
+        /// <param name="tasks">The task to iterate through when completed.</param>
         public static IAsyncEnumerable<Task> WhenEach(ReadOnlySpan<Task> tasks) => // TODO https://github.com/dotnet/runtime/issues/77873: Add params
```

Comment:
Oh no. We're missing params. This got missed.

Reply:
I guess `<inheritdoc` omits the parameters if the type is different, even if the name is the same.

Reply:
I meant we have a product bug: the signature should be `WhenEach(params ReadOnlySpan<Task> tasks)`

Reply:
https://github.com/dotnet/runtime/pull/107964
Probably doesn't meet the bar for .NET 9 at this point, unfortunately.

Reply:
Should we consider asking for an exception to fix the signature? I think we could even sneak it into RC2 if the branch is still open for the next day or two.

Reply:
I see, we should fix this.  Better to fix now before we ship it.

How many did we miss? https://github.com/dotnet/runtime/blob/5d8d04ec7b77299cdde6c3d069b0477f190cd121/src/libraries/System.Runtime/ref/System.Runtime.cs#L15884-L15889

Reply:
I'm aware of the two I fixed in the pr linked above. Still needs approval.

Reply:
I see from Stephen's PR, only 2 were missed: https://github.com/dotnet/runtime/pull/107964

I'm not inclined to do any heroics over this.  It's not breaking to add params.  We can just fix in 10.0